fixity/fixity.py

```diff
@@ -91,7 +95,7 @@ def fetch_environment_variables(namespace):
         namespace.report_user = namespace.report_pass = None
 
 
-def scan_message(aip_uuid, status, message):
+def scan_message(aip_uuid: uuid.UUID, status: bool, message: str) -> str:
```

Comment:
Why is `aip_uuid` typed as a `UUID`? The only call to `scan_message` I see is:

https://github.com/artefactual/fixity/blob/55e50db590ed937bd06553f56a1c0e13bed0d727/fixity/fixity.py#L171

And the docstring for `scan` documents it as string:

https://github.com/artefactual/fixity/blob/55e50db590ed937bd06553f56a1c0e13bed0d727/fixity/fixity.py#L128

Am I missing something?

Reply:
Nope, I need to change it.